scripts/upload-github-release-asset.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env sh
+#
+# Modified version of: https://gist.github.com/stefanbuck/ce788fee19ab6eb0b4447a85fc99f447
+#
+#
+# This script accepts the following parameters:
+#
+# * owner
+# * repo
+# * tag
+# * filename
+# * github_api_token
+#
+# Script to upload a release asset using the GitHub API v3.
+#
+# Example:
+#
+# upload-github-release-asset.sh github_api_token=TOKEN owner=stefanbuck repo=playground tag=v0.1.0 filename=./build.zip
+#
+
+set -e
+
+# Validate settings.
+[ "$TRACE" ] && set -x
+
+for line in "$@"; do
+  eval "$line"
+done
+
+# Define variables.
+GH_API="https://api.github.com"
+GH_REPO="$GH_API/repos/$owner/$repo"
+GH_RELEASES="$GH_REPO/releases"
+GH_TAGS="$GH_RELEASES/tags/$tag"
+AUTH="Authorization: token $github_api_token"
+
+if test "$tag" == "LATEST"; then
+  GH_TAGS="$GH_REPO/releases/latest"
+fi
+
+# Validate token.
+curl -o /dev/null -sH "$AUTH" $GH_REPO || { echo "Error: Invalid repo, token or network issue!";  exit 1; }
```

Comment:
Assuming empty token:

    curl -o /dev/null -sH "Authorization: token " https://api.github.com/repos/sass/node-sass && echo OK

This works, so no error here